pilot/pkg/networking/plugin/authz/rbac_test.go

```diff
@@ -558,6 +627,12 @@ func TestConvertRbacRulesToFilterConfig(t *testing.T) {
 			"service-role-2": policy2,
 		},
 	}
+	expectRbac3 := &policy.RBAC{
+		Action: policy.RBAC_ALLOW,
+		Policies: map[string]*policy.Policy{
+			"service-role-3": policy3,
```

Comment:
key of the map should be RoleBinding (service-role-binding-3). I realize the other test cases need to be updated too. @yangminzhu

Reply:
https://github.com/istio/istio/blob/master/pilot/pkg/networking/plugin/authz/rbac.go#L474 shows the key is the role name: rbac.Policies[role.Name] = &policyproto.Policy{...}

Reply:
@liminw The policy is generated based on `ServiceRole` and that's why we use the `ServiceRole`'s name as the policy's name. Otherwise I'm not sure how to handle the case when there are multiple `ServiceRoleBinding` for a single `ServiceRole`

Reply:
Ok. I see. I did not realize all the bindings for a role are merged to a single policy. In that case, using role name is correct. Thanks for explanation.